app/assets/javascripts/backbone/views/editable_table_view.js.coffee

```diff
@@ -0,0 +1,91 @@
+class MS.Views.EditableTableView extends Backbone.View
+
+  el: "table.editable-table"
```

Comment:
Perhaps a comment here reminding that this view may control multiple elements, so not to save any element-specific state.